osu.Game.Tests/Visual/Multiplayer/RoomManagerTestScene.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Game.Beatmaps;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Rulesets;
+using osu.Game.Screens.Multi;
+using osu.Game.Users;
+
+namespace osu.Game.Tests.Visual.Multiplayer
+{
+    public class RoomManagerTestScene : MultiplayerTestScene
```

Comment:
Shows up in the test scene right now - I suppose this was meant to be

```suggestion
    public abstract class RoomManagerTestScene : MultiplayerTestScene
```

right?